HTML/ex2/ex2.js

```diff
@@ -0,0 +1,26 @@
+
+
+function displayTime()
+{
+    const now = new Date();
+    const hours=now.getHours().toString().padStart(2.0);
+    const mins=now.getMinutes().toString().padStart(2,0);
+    const secs=now.getSeconds().toString().padStart(2,0);
+    const time=`${hours}:${mins}:${secs}`;
+    document.getElementById('clock').textContent=time;
+    
+
+
+}
+
+function toggle()
+{
+    var element = document.getElementById("container");
```

Comment:
Variable names can be better.
Don't scope variables globally if not required. Try to stick with `let` or `const`.